module.exports = (bot, error) => {
    return console.log(`Exception Caught!\n${error}`);
};